src/table/TableBodyWrapper.jsx

```diff
@@ -66,7 +78,7 @@ const TableBodyWrapper = ({ rootElement, tableData, constraints, selectionsAPI,
                   value={value}
                   key={column.id}
                   align={column.align}
-                  styling={styling}
+                  styling={{}}
```

Comment:
here the props - styling is assigned with {}. in withStyling.jsx, the styling gets CSS properties  but not in withSelections.jsx. why is it like that?

Reply:
So in the case of no column styling and no ongoing selections, there is not styling applied inline. When there is, that styling is added to `styling` in `withColumnStyling` and `withSelections`. Does that answer your question?

Reply:
yes. When there is, how is that styling added to `styling`?

Reply:
Just like before, in `styling-utils`. We need an empty object to start with, so that property always exist on the component, and then you can use any combination of `withColumnStyling` and `withSelections`.